<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoryWindow</class>
 <widget class="QWidget" name="CategoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1169</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>1101</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="g_deleteCategory">
    <widget class="QPushButton" name="search_4">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>300</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Chercher...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>240</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Saisissez l'id dte la catégorie que vous souhaitez supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="msg_4">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>280</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="categoryId_2">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>260</y>
       <width>42</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
    <widget class="QTableWidget" name="categoryTable_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>611</width>
       <height>531</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
    </widget>
    <widget class="QPushButton" name="deleteCategory">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>390</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="f_modifCategory">
    <widget class="Line" name="line_10">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>320</y>
       <width>641</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="search_3">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>280</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Chercher...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>220</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Saisissez l'id dte la catégorie que vous souhaitez modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modif_2">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>410</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="msg_3">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>270</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="categoryId">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>240</y>
       <width>42</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>370</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Catégorie: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>130</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier une catégorie</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="categoryTable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>551</width>
       <height>561</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
    </widget>
    <widget class="QLineEdit" name="categoryInput_2">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>360</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="e_addCategory">
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>180</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter une catégorie</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="categoryInput">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>230</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>230</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="addCategory">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>240</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="categoryTable_1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>561</width>
       <height>521</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="categoryComboBox_1">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>290</y>
       <width>201</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
